src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -570,6 +572,18 @@ def load_command_table(self, _):
         c.command('delete', 'delete')
         c.custom_command('set', 'server_dns_alias_set')
 
+    server_aadonly_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerAzureADOnlyAuthenticationsOperations.{}',
+        client_factory=get_sql_server_azure_ad_only_operations)
+
+    with self.command_group('sql server ad-only-auth',
+                            server_aadonly_operations,
+                            client_factory=get_sql_server_azure_ad_only_operations) as g:
+
+        g.custom_command('disable', 'server_aad_only_disable')
+        g.custom_command('enable', 'server_aad_only_enable')
+        g.command('get', 'get')
```

Comment:
CLI uses show command to do get operation. See  
https://github.com/Azure/azure-cli/blob/1ef59bace4dfad9cba137f49c8974aca4b0fff90/src/azure-cli/azure/cli/command_modules/sql/commands.py#L132